docs/schema/changelog.rst

```diff
@@ -18,6 +18,7 @@ ChangeLog
 Changed
 -------
 - The ``interestType`` and ``unspecifiedReason`` codelist codes have been changed from using hyphens to camelCase.
+- ``hasPepStatus`` and ``pepDetails`` are replaced with ``politicalExposure` object  that contains `status` and `details` properties.
```

Comment:
Backquotes need corrected to:

``hasPepStatus`` and ``pepDetails`` are replaced with ``politicalExposure`` object  that contains ``status`` and ``details`` properties.